Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: install install-dev install-pre-commit test style docs docs-serve
+
+install:
+	pip install sqlglot
```

Comment:
I don't think this is right.

Do we need this?

Reply:
I think I messed this one up, probably meant to do something like `pip install -e .`